src/camvidlog2/common/nms/torch.py

```
@@ -0,0 +1,64 @@
+    inter_y2 = torch.minimum(box[3], boxes[:, 3])
+
+    inter_w = (inter_x2 - inter_x1).clamp(min=0)
+    inter_h = (inter_y2 - inter_y1).clamp(min=0)
+    inter_area = inter_w * inter_h
+
+    area_box = (box[2] - box[0]) * (box[3] - box[1])
+    area_boxes = (boxes[:, 2] - boxes[:, 0]) * (boxes[:, 3] - boxes[:, 1])
+
+    union_area = area_box + area_boxes - inter_area
+    iou = torch.where(
+        union_area > 0, inter_area / union_area, torch.zeros_like(union_area)
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)